src/hooks/useIsAdult.js

```diff
@@ -0,0 +1,24 @@
+import { useNavigate } from 'react-router-dom';
+import { useEffect, useState } from 'react';
+import { saveToLocal, loadFromLocal } from '../utils/localStorage';
+
+export default function useIsAdult() {
+  const navigate = useNavigate();
+  const [isAdult, setIsAdult] = useState(loadFromLocal('isAdult') ?? false);
+  useEffect(() => {
+    saveToLocal('isAdult', isAdult);
+    isAdult ? navigate('./serien') : navigate('./');
+    // eslint-disable-next-line react-hooks/exhaustive-deps
```

Comment:
Not sure if you need it. Just wanted to mention it in case you missed it.

Reply:
Yes, i need it. It's a setting for ES-Lint :)